### PR TITLE
Text element style

### PR DESCRIPTION
**Pull request by maryam-mv**

### CODE REVIEW

File: website/src/components/pages/single-component/blocks-hub.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ const Bold = props => <strong css={{ fontWeight: 'bold' }} {...props} />;
 const Italic = props => <em css={{ fontStyle: 'italic' }} {...props} />;
 const Strike = props => <span css={{ textDecoration: 'strike-through' }} {...props} />;
 const Under = props => <span css={{ textDecoration: 'underline' }} {...props} />;
+const leftIndent = {
+	paddingLeft: '5rem',
```
Comment (MadeByMike) [inline website/src/components/pages/single-component/blocks-hub.js]: Can we use the spacing helpers?

File: website/src/components/pages/single-component/blocks-hub.js [hunk 2]
```diff
@@ -12,6 +12,12 @@ const Bold = props => <strong css={{ fontWeight: 'bold' }} {...props} />;
...
+	'@media (max-width: 568px)': {
+		paddingLeft: '0rem',
```
Comment (MadeByMike) [inline website/src/components/pages/single-component/blocks-hub.js]:
```suggestion
		paddingLeft: 0,
```

File: website/src/components/pages/single-component/blocks-hub.js [hunk 3]
```diff
@@ -12,6 +12,12 @@ const Bold = props => <strong css={{ fontWeight: 'bold' }} {...props} />;
 const Italic = props => <em css={{ fontStyle: 'italic' }} {...props} />;
 const Strike = props => <span css={{ textDecoration: 'strike-through' }} {...props} />;
 const Under = props => <span css={{ textDecoration: 'underline' }} {...props} />;
+const leftIndent = {
+	paddingLeft: '5rem',
+	'@media (max-width: 568px)': {
```
Comment (MadeByMike) [inline website/src/components/pages/single-component/blocks-hub.js]: Are breakpoints in the design tokens?

File: website/src/components/pages/single-component/blocks-hub.js (hunk 1 above)
Comment (dominikwilkowski) [reply website/src/components/pages/single-component/blocks-hub.js]: hah I missed those. nice catch

File: website/src/components/pages/single-component/blocks-hub.js (hunk 3 above)
Comment (dominikwilkowski) [reply website/src/components/pages/single-component/blocks-hub.js]: Yes but that doesn't mean you can add breakpoints anywhere where it makes sense

**Pull request merged**